managed/ui/src/components/certificates/Certificates.js

```diff
@@ -116,8 +118,61 @@ class Certificates extends Component {
       .finally(() => this.setState({ showSubmitting: false }));
   };
 
+  /**
+   * Delete the root certificate if certificate is safe to remove,
+   * i.e - Certificate is not attached to any universe for current user.
+   * 
+   * @param certificateUUID Unique id of certificate.
+   */
+  deleteRootCertificate = (certificateUUID) => {
+    api.deleteCertificate(certificateUUID).then(
+      () => {
+        this.getCertificateArray()
+      }
+    ).catch(
+      err => {
+        console.log("Error", err)
+      }
+    )
+  };
+
+  /**
+   * Lifecycle method to fetch iniial data required by compoenent
+   * i.e certificates attached to current user.
+   */
+  componentDidMount() {
+    this.getCertificateArray();
+  }
+
+  /**
+   * Fetch certificates attched to current user.
+   */
+  getCertificateArray = () => {
+    api.getCertificates().then(
+      response => {
+        const certificateArray  = response
+        ? response.map((cert) => {
+          return {
+            type: cert.certType,
+            uuid: cert.uuid,
+            name: cert.label,
+            expiryDate: cert.expiryDate,
+            certificate: cert.certificate,
+            creationTime: cert.startDate,
+            privateKey: cert.privateKey,
+            customCertInfo: cert.customCertInfo,
+            inUse: cert.inUse
+          };
+        })
+        : [];
+      this.setState({ certificateArray: certificateArray})
```

Comment:
```suggestion
      this.setState({ certificateArray });
```
and actually no need to store certs list in the local state as it comes as prop from the redux store

Reply:
Done